test/tests/mortar/displaced-gap-conductance-2d-auxkernel/tests

```diff
@@ -0,0 +1,13 @@
+[Tests]
+  design = 'Constraints/index.md'
+  issues = '#13080'
+  [gap_conductance]
+    type = Exodiff
+    input = 'gap-conductance.i'
+    exodiff = 'gap-conductance_out.e'
+    custom_cmp = 'auxkernel.cmp'
+    mesh_mode = 'REPLICATED'
+    max_parallel = 2
+    requirement = 'We shall be able to couple variables between auxiliary kernels and ad mortar constraints'
```

Comment:
Remove the "ad" portion of this requirement, and add a period at the end of the sentence

Reply:
Also maybe say "boundary auxiliary kernels" or do we actually also support block aux kernels with mortar constraints? I'd have to think about that one

Reply:
You are right. It should be "boundary auxiliary kernels" here. Not sure if block-wise aux kernels are supported with Mortar constraints. I'd say probably not since it has not been tested yet.